src/main/java/com/tickets/util/InjectData.java

```diff
@@ -0,0 +1,40 @@
+package com.tickets.util;
+
+import com.tickets.model.Role;
+import com.tickets.model.Roles;
+import com.tickets.model.User;
+import com.tickets.service.model.RoleService;
+import com.tickets.service.model.ShoppingCartService;
+import com.tickets.service.model.UserService;
+import java.util.List;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
+
+@Component
+public class InjectData {
+    private final RoleService roleService;
+    private final UserService userService;
+    private final ShoppingCartService shoppingCartService;
+
+    public InjectData(RoleService roleService, UserService userService,
+                      ShoppingCartService shoppingCartService) {
+        this.roleService = roleService;
+        this.userService = userService;
+        this.shoppingCartService = shoppingCartService;
+    }
+
+    @PostConstruct
+    public void init() {
+        Role admin = new Role();
+        admin.setRoles(Roles.ADMIN);
+        Role user = new Role();
+        user.setRoles(Roles.USER);
+        roleService.add(admin);
+        roleService.add(user);
+        User userAdmin = new User();
+        userAdmin.setEmail("admin@example.com");
+        userAdmin.setPassword("1234");
```

Comment:
```suggestion
        userAdmin.setPassword("1234");
```
to login by admin user we should encode() our password too

Reply:
@Fedich09 password is encoding in UserService)